src/amo/types/categories.js

```diff
@@ -13,6 +13,6 @@ export type CategoryType = {|
     | 'dictionary'
     | 'search'
     | 'language'
-    | 'persona',
+    | 'statictheme',
```

Comment:
I'm not sure about this one. I notice that `theme` is still listed there, and that's not really valid any longer either. We really only have `statictheme` now, correct? So maybe both `theme` and `persona` should be removed and replaced with just `statictheme`?

Reply:
ah yes, correct